rust/src/ike/logger.rs

```diff
@@ -73,14 +73,19 @@ fn log_ike(
     }
 
     if tx.ike_version == 1 {
-        let mut index = 0;
-        for server_transform in &state.ikev1_container.server.transforms {
-            if index >= 1 {
-                debug_validate_bug_on!(true);
-                break;
+        if state.ikev1_container.server.transforms.len() > 0 {
+            // log the first transform as the chosen one
+            add_attributes(&state.ikev1_container.server.transforms[0], jb)?;
+        }
+        if state.ikev1_container.server.transforms.len() > 1 {
+            // in case we have multiple server transforms log them in a list
+            jb.open_array("server_proposals")?;
```

Comment:
I wonder if we should have an S-V test for this

Reply:
Maybe the doc should be updated as well...

Reply:
> I wonder if we should have an S-V test for this

Not sure about that, at least it seems to be a rare case. (not in the rfc and the first pcap I saw with this property)